File: Coding Club Beginner Projects/Lesson2/Calculator/calculator.py
num1 = float(input("Please enter in the first number: "))

operation = input("Please enter in the operation (+, -, *, /, %): ")

num2 = float(input("Please enter in the second number: "))

result = 0

if operation == "+":
    result = num1 + num2
elif operation == "-":
    result = num1 - num2
elif operation == "*":
    result = num1 * num2
elif operation == "/":
    result = num1 / num2
elif operation == "%":
    result = num1 % num2
else:
    result = "Invalid operation entered..."

print("The result is: " + str(result))

File: Coding Club Beginner Projects/Lesson2/Calculator/calculator.js
var num1 = prompt("Please enter in the first number: ") * 1;

var operation = prompt("Please enter in the operation (+, -, *, /, %): ");

var num2 = prompt("Please enter in the second number: ") * 1;

var result = 0;

if (operation == '+')
{
    result = num1 + num2;
}
else if (operation == '-')
{
    result = num1 - num2;
}
else if (operation == '*')
{
    result = num1 * num2;
}
else if (operation == '/')
{
    result = num1 / num2;
}
else if (operation == '%')
{
    result = num1 % num2;
}
else
{
    result = "Invalid operation entered...";
}

console.log("The result is: " + result);